Mathlib/Algebra/Polynomial/Derivative.lean

```diff
@@ -126,6 +126,10 @@ theorem derivative_sum {s : Finset ι} {f : ι → R[X]} :
     derivative (∑ b ∈ s, f b) = ∑ b ∈ s, derivative (f b) :=
   map_sum ..
 
+theorem iterate_derivative_sum {s : Finset ι} {f : ι → R[X]} {k : ℕ} :
+    derivative^[k] (∑ b ∈ s, f b) = ∑ b ∈ s, derivative^[k] (f b) := by
```

Comment:
```suggestion
theorem iterate_derivative_sum (k : ℕ) (s : Finset ι) (f : ι → R[X]) :
    derivative^[k] (∑ b ∈ s, f b) = ∑ b ∈ s, derivative^[k] (f b) := by
```
because it's a rewriting lemma, and to match the order in which the variables appear

Reply:
I think it's OK to make the parameters implicit? Just like what `derivative_sum` does.

Reply:
I think it's not ok, no. This is a rewriting lemma. It should have its main arguments explicit. `derivative_sum` should be changed too